libs/components/src/anon-layout/anon-layout-wrapper.component.html

```diff
@@ -7,6 +7,7 @@
   [hideCardWrapper]="hideCardWrapper"
   [hideBackgroundIllustration]="hideBackgroundIllustration"
 >
+  <router-outlet slot="headerActions" name="headerActions"></router-outlet>
```

Comment:
This should probably be dash based to follow the usual convention we have.
```suggestion
  <router-outlet slot="header-actions" name="header-aactions"></router-outlet>
```